wagtail_localize/wagtail_hooks.py

```diff
@@ -148,8 +160,9 @@ def page_listing_more_buttons(page, page_perms, next_url=None):
         if has_locale_to_translate_to:
             url = reverse("wagtail_localize:submit_page_translation", args=[page.id])
 
-            yield wagtailadmin_widgets.Button(
-                _("Translate this page"), url, priority=60
+            yield set_button_icon(
+                wagtailadmin_widgets.Button(_("Translate this page"), url, priority=60),
+                "site",
```

Comment:
Icon ok?

Reply:
Wow, that was easy, but please check the svg code. I've used the "solid" version of the "language"-icon from https://heroicons.com/. It looks good to me, but I'm not a frontend-guy ;)
![wagtail_localize_language_icon](https://user-images.githubusercontent.com/468164/194139678-610ec550-44b6-4e3a-9f6f-ca0cc87c5274.png)

Reply:
The icon looks good to me locally 👍